modules/tinymce/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `color_default_foreground` and `color_default_background` options to set the initial default color for the `forecolor` and `backcolor` toolbar buttons and menu items. #TINY-9183
 - New `getTransparentElements` function added to `tinymce.html.Schema` to return a map object of transparent HTML elements. #TINY-9172
 - Added `ToggleToolbarDrawer` event to subscribe to toolbar's opening and closing. #TINY-9271
+- Added `typography` premium plugin to the premium plugin list. #TINY-9298
```

Comment:
I don't think we usually add these to the core changelog, if you look at the rest of the file

- [ ] revert